packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts

```diff
@@ -0,0 +1,47 @@
+// @group:suite
+// @retry=2
+
+describe('Go into new accounts', () => {
```

Comment:
```suggestion
describe('New accounts', () => {
```
Let's use this name as name of test case

Reply:
534889de5464e01a3eb5970394719506dc079b1e done